package.json

```diff
@@ -36,7 +36,7 @@
     "prebuild": "rimraf dist && node scripts/gen-version.js",
     "build": "tsc && rollup -c rollup.config.ts",
     "start": "tsc -w",
-    "test": "jest",
+    "test": "jest --forceExit",
```

Comment:
Hrm. This smells like there's something wrong with the tests... async operations not resolving?

Reply:
Yep. I tried to track it down for a while, but gave up. From my testing it seems like in some edge-cases it's possible that ipfs leaves a child process and some sockets hanging after `await ipfs.stop()`. Not sure what's up with that, but after fighting with it for a while I just gave up.
The IPFS we're using in the tests is basically in-memory only. It doesn't write to disk. So killing it shouldn't be an issue.

Reply:
Cool. yeah, something we can track down and fix as a follow up later, imo.